src/query/__tests__/expression.test.ts

```
@@ -0,0 +1,243 @@
+import { describe, it, expect } from 'vitest';
+import { condition, toString, and, or, not } from '../expression.js';
```

Biome (2.1.2)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)